packages/amesos2/src/Amesos2_Superlu_FunctionMap.hpp

```diff
@@ -805,10 +815,11 @@ namespace Amesos2 {
     }
 
     static void create_CompCol_Matrix(SLU::SuperMatrix* A, int m, int n, int nnz,
-				      SLU::C::complex* nzval, int* rowind, int* colptr,
+				      void * nzval, int* rowind, int* colptr,
```

Comment:
Did SuperLU interface change ? Then do we have support older SuperLU ?

Reply:
It did not change but I thought this might be the easiest place to store the reinterpret_cast step. I could restore the native type here and have the solver cast the ptr before calling the FunctionMap. Resolving this is probably related to the 1st comment and possible problems with how I did TypeMap changes.